### DISCUSSION TITLE
What is the release process/schedule?

### DISCUSSION THREAD

**Discussion by ynagasaki** [mqttjs/MQTT.js#1620] [Q&A]
Hello, just wondering what the release process/schedule is like. This PR (https://github.com/mqttjs/MQTT.js/pull/1151) was recently merged, and I'd like to know when/if it will be available from NPM.

Apologies if this is covered elsewhere, please feel free to forward me the link.

Thanks!

**Comment by BearCooder** [comment]
@robertsLando can probably tell more but I guess it will be added to a new v5 beta release soon.
See: https://github.com/mqttjs/MQTT.js/issues/1541#issuecomment-1609618153

**Reply by ynagasaki** [reply]
I see; if possible, it would be nice to get a v4 release out. If v5 is in beta and has breaking changes, then it will be harder to include the change in our project (as we'd have to wait until v5 is out of beta and then test the upgrade).

**Reply by robertsLando** [reply]
we just dropped EOL nodejs version so the upgrade should be painless.

**Reply by ynagasaki** [reply]
@robertsLando I just tried out the `5.0.0-beta.2` release, and the minified JS that's bundled with the release package (i.e. `dist/mqtt.min.js`) contains a lot of more modern syntax that is causing a lot of problems for our project's babel setup (which is admittedly a bit old), namely:

* object spread operator
* async function generators
* BigInt literals and math
 
None of these language features were used in `v4`'s `mqtt.min.js` and I'm not sure what changed or introduced them between the last v4 release and v5? What bundler do you use to generate `mqtt.min.js` and did you change it recently?

I might be able to upgrade our babel version and that might fix these issues, but I also just wanted to get a better understanding of what happened. Thanks.

**Reply by robertsLando** [reply]
@ynagasaki Thanks for your feedback. Anyway if we want to renew MQTTjs I think it makes sense to drop some old style syntax in favor of newer and cleaner one :) So please consider updating babel 🙏🏼

**Comment by robertsLando** [comment]
Oh this is bad. Just noticed that the PR has ben merged on master but now the branch is named `main` :(

**Comment by robertsLando** [comment]
@ynagasaki I have created https://github.com/mqttjs/MQTT.js/pull/1622 to move all master commits to main.
